frontend/src/app/home/model/office.model.ts

```diff
@@ -0,0 +1,35 @@
+import { DetailList } from './detail-list.model';
+import { MapObject } from './map-object.model';
+
+export class Office extends MapObject {
+  firstName: string;
+  lastName: string;
+  number: string;
+  position: string;
+  isCoordinator: boolean;
+  team: string;
+  isVolunteer: boolean;
+
+  constructor(office: Office) {
+    super(office);
+    this.firstName = office.firstName;
+    this.lastName = office.lastName;
+    this.number = office.number;
+    this.position = office.position;
+    this.isCoordinator = office.isCoordinator;
+    this.team = office.team;
+    this.isVolunteer = office.isVolunteer;
```

Comment:
tego typu przypisania zazwyczaj można skrócić, więc jak kiedyś coś takiego wam się pojawi to najlepiej poszukać coś o tym np tu powinno zadziałać np. 
```
  constructor(office: Office) {
    super(office);
    Object.assign(this, office);
  }
```